dockstore-webservice/src/main/resources/migrations.1.9.0.xml

```diff
@@ -97,4 +97,10 @@
             <column name="os" type="varchar(255 BYTE)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet author="esoth (generated)" id="1586206601202-8">
+        <addColumn tableName="enduser">
+            <column name="orcid" type="varchar(255 BYTE)"/>
```

Comment:
No real performance difference gain when using length-constrained column.
Unlikely to be attack vector either
See tip https://www.postgresql.org/docs/9.3/datatype-character.html